MAINTAINERS.md

```diff
@@ -0,0 +1,90 @@
+# Maintainers
+
+The current technical steering committee (TSC) are also maintainers of this repo. More information can be found [here](https://github.com/hiero-ledger/tsc).
+
+## Active Maintainers
+
+| Name            |	GitHub ID	                                   | GitHub Scope |	LFID  |	Discord ID  |	Email    | 
+|-----------------|----------------------------------------------|--------------|-------|-------------|----------|
```

Comment:
It would be nice to get this lined up (also it looks like there may be tabs in here)